src/main/java/it/unisa/Amigo/documento/service/DocumentoServiceImpl.java

```diff
@@ -108,17 +111,25 @@ public Resource loadAsResource(Documento documento) {
      * @return documento con id uguale a idDocumento.
      */
     @Override
-    public Documento findDocumento(int idDocumento) {
+    public Documento findDocumento(Integer idDocumento) {
```

Comment:
```suggestion
    public Documento findDocumentoById(Integer idDocumento) {
```
Quando applicate il cambio nome usare la funzionalità di refactoring su intellij, non cambiate il nome a mano